<!-- DESCRITPION -->

<launch>
	<node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger">
	<param name="destination_frame" value="scan_front"/>
	<param name="cloud_destination_topic" value="/merged_cloud"/>
	<param name="scan_destination_topic" value="/scan_multi"/>
	<param name="laserscan_topics" value ="/scan_front /scan_mid_mid /scan_mid_right /scan_mid_left" /> <!-- LIST OF THE LASER SCAN TOPICS TO SUBSCRIBE -->
	<param name="angle_min" value="-3.5"/>
	<param name="angle_max" value="3.5"/>
	<param name="angle_increment" value="0.005759587"/>
	<param name="scan_time" value="0.000061722"/> <!--changed 28.06.23 - from 0.0333333 to 000061722 -->
	<param name="range_min" value="0.05"/>
	<param name="range_max" value="5.0"/>
	</node>
</launch>
